### PR TITLE
Feature/landing page

### PR DESCRIPTION
**Pull request by joanafbrito**
## Implemented Features:

-Started App And Add Routers
- Started Landing Page
- Started Search Component

## Roadblocks/Logs/Errors:

- We still need to work on the Logic, just did a starting point

## Testing:

- Used /localhost3000 and devTools

## Notes/For the future:

- We have to add logic into the Search component, 
-Need to decide on the images 
- We have  to work on the bar to be on the top of the image 

## Resources:

- Google hehe

### CODE REVIEW

**Review by Jtpiland — COMMENTED** [review]
This PR looks good to me. Travis build seems to be failing, I tried to follow the error message but it looked a little like wingdings to me... happy to try and troubleshoot that with somebody whenever!

File: src/components/Landing/Landing.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import './Landing.css';
+
+
+const Landing = () => {
+    return (
+        <section className='landing'>
+            <img className='background-img'src='https://thumbs.dreamstime.com/b/ocher-apron-various-baking-ingredients-eggs-sugar-cinnamon-sticks-kitchen-utensils-white-background-top-view-copy-175923956.jpg' alt='baking'/>
+                <div>
+                    <img className='logo'src="" alt="LoHi logo"/>
+                    <h1>Where your AlTituDe matters!</h1>
```
Comment (Meggs625) [inline src/components/Landing/Landing.js:12]: Best tag line!

File: src/components/Search/Search.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
+.search-field {
```
Comment (Meggs625) [inline src/components/Search/Search.css:1]: Love that you already jumped into that styling. Joana style 💞

Comment (joanafbrito) [reply src/components/Search/Search.css:1]: Yes, my favorite searching type hehe

**Pull request merged**